packages/asset-uploader/src/utils/filter-by-type.ts

```diff
@@ -1,12 +1,12 @@
+import { FONT_FORMATS } from "@webstudio-is/fonts";
 import type { AssetType } from "../types";
-import { FONT_FORMATS } from "../constants";
 
 export const filterByType = <PartialAsset extends { format: string }>(
   assets: Array<PartialAsset>,
   type: AssetType
 ) => {
   return assets.filter(({ format }: PartialAsset) => {
-    const isFont = FONT_FORMATS.includes(format as typeof FONT_FORMATS[number]);
+    const isFont = format in FONT_FORMATS;
```

Comment:
Looks like a bug because:

```
'ttf' in ['ttf']
> false
```

Same in some other places if you search by `in FONT_FORMATS`

Reply:
FONT_FORMATS is an object, I think you looked at the previous state where it was an array

Reply:
actually changed it now to use Map, isn't much of importance but is nicer